virtio-devices/src/vhost_user/fs.rs

```diff
@@ -315,27 +320,31 @@ impl Fs {
 
         // Identify if protocol features are supported by the slave.
         let mut acked_features = 0;
+        let mut acked_protocol_features = 0;
         if avail_features & VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits() != 0 {
             acked_features |= VhostUserVirtioFeatures::PROTOCOL_FEATURES.bits();
 
             let mut protocol_features = master
                 .get_protocol_features()
                 .map_err(Error::VhostUserGetProtocolFeatures)?;
 
+            let mut protocol_features_mask = VhostUserProtocolFeatures::MQ
```

Comment:
Not sure if this should really be called a mask as that unusual involves inverted bits. `supported_protocol_features` would be better.

Reply:
I agree, let me update that.